interface/src/app/analyze/analysis/sampleBin.tpl.html

```diff
@@ -1,10 +1,9 @@
-<svg id="sample-bin" width="100" height="60" ng-click="show()">
-  <image x="27" y="6" width="45" height="45"
-      xlink:href="static/assets/testtube-expt2.svg">
-  </image>
-  <g ng-if="sb.heatmapData.samples.length">
-    <text text-anchor="left" x="28" y="43">&times;</text>
-    <text text-anchor="left" x="44" y="43">{{
-      sb.heatmapData.samples.length }}</text>
-  </g>
-</svg>
+<li ui-sref-active="active" class="sample-bin">
+  <a href ui-sref="analysis-detail">
```

Comment:
Sure enough. Taking out `href` works fine. Angular will stick it back in when processing the `ui-sref` property as described in the `ui-router` docs referenced by @dongbohu above. Thank you!